lib/control-app.js

```diff
@@ -30,9 +31,13 @@ module.exports = function(config) {
         fullPaths: true
     });
 
-    files = files.map(function(file) {
-        return path.resolve(file);
+    var expandedFiles = [];
+    files.forEach(function (arg) {
```

Comment:
Could also use map.

I'd rather change this into one line of tested code using [globs-to-files](https://github.com/chinedufn/globs-to-files/blob/master/index.js#L33-L36) and then `var files = deglob.sync(files)`

But let me know what you prefer and I'll do it :)